packages/admin/src/pages/reviewers/Form/statusForm.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+	SimpleForm,
+	SelectInput,
+	useNotify,
+	useRedirect,
+	useDataProvider,
+	SaveButton,
+	Toolbar,
+	ReferenceField,
+	TextField,
+	required,
+} from 'react-admin';
+
+const StatusForm = ({ record, resource, basePath }) => {
+	const [status, setStatus] = useState(record.status);
+	const [loading, setLoading] = useState(true);
+
+	const CustomToolbar = () => {
+		const notify = useNotify();
+		const redirect = useRedirect();
+		const dataProvider = useDataProvider();
+		const submit = () =>
+			dataProvider
+				.update(`${resource}/${record.id}/update-status`, { id: '', data: { status } })
+				.then(() => {
+					notify('ra.notification.updated', 'info', { smart_count: 1 });
+					redirect('list', basePath);
+				})
+				.catch(() => {
+					notify('ra.notification.http_error', 'warning');
+					setLoading(false);
+				});
+
+		const handleSubmit = () => {
```

Comment:
I think we can remove this function, cause, it, actually, turns the code even bigger 😄 . You can just use `setLoading(true)` in the submit function 👍